azure-pipelines.yml

```diff
@@ -38,7 +38,7 @@ resources:
   containers:
   # Container with Maven 3.2.5, SSL to have the same environment everywhere.
```

Comment:
A link to the repository as the comment would be good enough, I guess.

Reply:
I would keep the comment in any case, because it is way easier to understand than having to dig through a dockerfile in a separate repo.

Reply:
It does admittedly duplicate information, but that's fine I think.